test/__mocks__/text.mock.ts

```diff
@@ -142,6 +142,7 @@ export const UPDATE_OVERSEAS_ENTITY_PAGE_TITLE = "Confirm or update overseas ent
 export const CONFIRM_AND_CONTINUE_BUTTON_TEXT = "Confirm and continue";
 export const UPDATE_OVERSEAS_ENTITY_TITLE = "Update overseas entity";
 export const OE_NUMBER_FIELD_POPULATED = "value=\"OE123456\"";
+export const UPDATE_CHECK_YOUR_ANSWERS_PAGE_TITLE = "Check your answers";
```

Comment:
I think moving this from line 153 is unnecessary, save yourself 2 line changes :D

Reply:
@corybudd-ch resolved